sklearn/preprocessing/data.py

```diff
@@ -1995,8 +2004,13 @@ def fit(self, K, y=None):
                              .format(K.shape[0], K.shape[1]))
 
         n_samples = K.shape[0]
+        if K.shape[1] != n_samples:
+            raise ValueError(
```

Comment:
is this tested?

Reply:
hm this shouldn't be here any more, it was fixed in a different PR.